test/analysis_ee_Z_bb_cfg.py

```diff
@@ -29,25 +29,50 @@
 Collider.BEAMS = 'ee'
 Collider.SQRTS = 91.
 
+from heppy.configuration import PapasRuntype
+
 # input definition
 ee_Z_bbbar = cfg.Component(
     'ee_Z_bbbar',
-    files = ['/'.join([os.environ['HEPPY'],
-                              'test/data/ee_Z_bbbar_with_papas_rec.root'] )]
+    files = [
+        'data/ee_Z_ddbar.root'
+    ]
 )
 
 selectedComponents = [ee_Z_bbbar]
 
-# read FCC EDM events from the input root file(s)
-# do help(Reader) for more information
-from heppy.analyzers.fcc.Reader import Reader
-source = cfg.Analyzer(
-    Reader,
-    gen_particles = 'GenParticle',
-    gen_vertices = 'GenVertex'
-)
+if PapasRuntype.from_fccsw :
```

Comment:
setup a specific cfg module for this sequence, on the model of papas_cfg

Reply:
Done